app/src/main/java/ch/sdp/vibester/activity/SearchUserActivity.kt

```diff
@@ -73,12 +92,19 @@ class SearchUserActivity : AppCompatActivity() {
         userProfileAdapter.notifyDataSetChanged()
     }
 
+    /**
+     * Callback to update users in adapter during search
+     */
+    private fun setUserInAdapter2(users: ArrayList<String> = ArrayList()) {
+        uidList = ArrayList(users)
```

Comment:
why do you need to wrap users in ArrayList? They are already, no?

Reply:
Here this is only the uid. Do you think it is better to do another way ?